src/factorizations.jl

```diff
@@ -1,25 +1,42 @@
 using LinearAlgebra: LinearAlgebra
-using MatrixAlgebraKit:
-  eig_full!,
-  eig_trunc!,
-  eig_vals!,
-  eigh_full!,
-  eigh_trunc!,
-  eigh_vals!,
-  left_null!,
-  left_orth!,
-  left_polar!,
-  lq_full!,
-  lq_compact!,
-  qr_full!,
-  qr_compact!,
-  right_null!,
-  right_orth!,
-  right_polar!,
-  svd_full!,
-  svd_compact!,
-  svd_trunc!,
-  svd_vals!
+using .MatrixAlgebra: MatrixAlgebra
```

Comment:
```suggestion
```
This is already included in `TensorAlgebra.jl`.
I think it could be clearer to move all imports to that file, to avoid these kinds of things.